.travis.yml

```diff
@@ -1,8 +1,13 @@
 os: linux
-language: go
-go_import_path: github.com/operator-framework/operator-sdk
 dist: xenial
 
+# Install python3 and utilities dependencies
+addons:
+  apt:
+    packages:
+      - "python3"
+      - "python3-pip"
```

Comment:
Probably better for a follow-on, but we could shave off around 10-15s (it looks like) from most build jobs if we moved these package installs to the ansible e2e job only.